### ISSUE TITLE
Would it be possible to have a example showing how to manage multiple pages

### ISSUE THREAD

**Issue by mardorfv** [matejsvajger/scffld#3]
I've been using SCFFLD to setup marketing landing pages. It would be much better if I could organize them into a site. Thank you for the project template I really enjoy using it.

**Comment by matejsvajger** [comment]
Hey, nice to see someone else using scffld!

You could organize your code so you would have multiple entry points.

Each *.pug file directly in `/src` folder is already compiled to its' own html file. These pug files could extend a required layout for the page and in `_styles` you could setup multiple css entry points to build a separate css file for each site. This way you would have control over the styles for each site, but still would be able to re-use modules across the sites.

Something like this:

```
├── src
|   ├── _layouts
|   |   └── base.pug
|   |   └── full-width.pug
|   |   └── sidebar-left.pug
|   |   └── sidebar-right.pug
|   ├── _styles
|   |   └── page-one
|   |      └── main.sass    
|   |   └── page-two
|   |      └── main.sass    
|   └── page-one.pug             # Page one template
|   └── page-two.pug             # Page two template
```
and in config.json:
```json
"styles": {
    "src": "_styles/page-*",
    "dist": "css"
},
```

Is this what you're trying to achieve?

**Comment by mardorfv** [comment]
Yes I'll try this setup. I use scffld all the time with material design
components.

Thanks!

**Comment by mardorfv** [comment]
How could I change settings so the site compiles more then one scss file? I'm trying to load a grid.css and main.css into my pages.

**Comment by mardorfv** [comment]
I changed the style css entries in both package and config json files to be "**" , that seemed to work.

**Comment by mardorfv** [comment]
Is it possible to do same with javascript? Have multiple main.js files for different layouts. Im working on 2 different projects rebranding the same front-end framework for both.
